scintellometry/io/filehandlers.py

```diff
@@ -22,8 +22,8 @@
 # size in bytes of records read from file (simple for ARO: 1 byte/sample)
 def dtype_itemsize(dtype):
     bps = {'ci1': 2, '(2,)ci1': 4, 'ci1,ci1': 4,
-           '1bit': 0.125, '2bit': 0.25, '4bit': 0.5, 'c4bit': 1,
-           'cu4bit,cu4bit': 2}.get(dtype, None)
+           '1bit': 0.125, '2bit': 0.25, '4bit': 0.5, '4bit,4bit': 1,
```

Comment:
I think here you would simply add `'(2048,)4bit': 1024` (and use that for `dtype` in the AROCHIMEInvPFB class).